src/AppEndToEnd.test.tsx

```diff
@@ -170,7 +173,14 @@ it('VxMark+Print end-to-end flow', async () => {
     // Vote for candidate contest
     if (title === presidentContest.title) {
       fireEvent.click(getByText(presidentContest.candidates[0].name))
+
+      // advance time by a second, run timers, and see if things get saved
+      advanceBy(1100)
+      advanceTimers()
+
+      expect(fetchMock.calls('/card/write_long_b64')).toHaveLength(1)
```

Comment:
There should be tests to advance time and expect no change in fetchMock.calls too.